sensor/common/clusterentities/store.go

```diff
@@ -98,8 +97,8 @@ func (e *Store) initMaps() {
 }
 
 func (e *Store) resetMaps() {
-	e.historyMutex.Lock()
-	defer e.historyMutex.Unlock()
+	e.mutex.Lock()
+	defer e.mutex.Unlock()
```

Comment:
I think this is going to deadlock. `resetMaps` is called from the `Cleanup` function that also locks this `mutex`.

Reply:
should be addressed in [be06eac](https://github.com/stackrox/stackrox/pull/13155/commits/be06eace6157f2fd90b7d172664124c36e63c6f3)